src/components/Global/TokenSelector/Components/AdvancedButton.tsx

```
@@ -131,16 +133,32 @@ export const AdvancedTokenSelectorButton = ({
+                                <Icon
+                                    name={'arrow-bottom'}
+                                    className={`h-12 w-12 transition-transform dark:fill-white ${isVisible ? 'rotate-180 ' : ''}`}
+                                />
+                        </div>
+                    }
+                    {'xchain' === type &&
+                        <div
+                            className="block"
+                            onClick={(e) => {
+                                e.stopPropagation()  //don't open modal
+                                onReset && onReset()
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)